specs/latest/2.0/index.html

```diff
@@ -1457,21 +1557,28 @@ <h2 class="no-toc">Table of contents</h2>
         </p>
       </dd>
 
-      <dt class="idl-code"><a name="TEXSUBIMAGE2D">void texSubImage2D</a>(GLenum target, GLint level, GLint xoffset, GLint yoffset, GLsizei width, GLsizei height, GLenum format, GLenum type, ArrayBufferView? pixels)
+      <dt class="idl-code"><a name="TEXSUBIMAGE2D">
+        void texSubImage2D</a>(GLenum target, GLint level, GLint xoffset,
+                               GLint yoffset, GLsizei width, GLsizei height,
+                               GLenum format, GLenum type,
+                               BufferDataSource srcData,
```

Comment:
Let's make this "BufferDataSource?" so that this continues to follow the OpenGL style of generating errors -- rather than a DOMException during the call because the argument isn't nullable.